db/migrate/20230607202616_create_posts.rb

```diff
@@ -1,13 +1,13 @@
 class CreatePosts < ActiveRecord::Migration[7.0]
   def change
     create_table :posts do |t|
-      t.string :Title
-      t.text :Text
-      t.integer :CommentsCounter
-      t.integer :LikesCounter
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, foreign_key: true
 
```

Comment:
- [ ] It would be nice if you update this migration as well because in the post table we need author_id which will help us in linking the user and post 👍